ilastik/config.py

```diff
@@ -63,25 +132,36 @@ class RuntimeCfg:
     preferred_cuda_device_id: Optional[str] = None
 
 
-cfg: configparser.ConfigParser = configparser.ConfigParser()
+cfg: IlastikPreferences = IlastikPreferences()
 cfg_path: Optional[Path] = None
 runtime_cfg: RuntimeCfg = RuntimeCfg()
 
 
-def _init(path: Union[None, str, bytes, os.PathLike]) -> None:
+def _init(path: Union[str, bytes, os.PathLike]) -> None:
     """Initialize module variables."""
-    config_path = Path(path) if path is not None else None
+    config_path = Path(path)
+
+    if config_path.is_file():
+        logger.info(f"Loading configuration from {config_path!s}.")
```

Comment:
Note that this will not actually be logged unless the config file path is specified with `--configfile=` (i.e. init called through `app.main()`, not during a regular load from ilastik.ini in the default location.

I think that's fine (and not a consequence of this PR anyway), but the import-time init in this module isn't a great choice for tracing what init happens when and where.